### PR TITLE
More snippet/flier entries for mods

### PR DESCRIPTION
**Pull request by DangerNoodle**
Added snippet entries for Animatronic Monsters, DinoMod, DeadLeaves' Fictional Guns, Medieval and Historic Content, More Survival Tools, and Tanks and Other Vehicles.

### CODE REVIEW

File: data/mods/More_Survival_Tools/snippets.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "type": "snippet",
+    "category": "note",
+    "//": "Random graveyard epitaph from Nethack, and also referencing prehistoric hunter profession in the mod.",
```
Comment (BorkBorkGoesTheCode) [inline data/mods/More_Survival_Tools/snippets.json]: Might need attribution

File: data/mods/Tanks/snippets.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Next person to call this infantry fighting vehicle a \"tank\" is walking home.\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Found what used to be an armored platoon.  Most tanks have hatches up top, not in the back like the new ones they're using.\"" ]
+  }
```
Comment (BorkBorkGoesTheCode) [inline data/mods/Tanks/snippets.json]: Merkava

Comment (DangerNoodle) [inline data/mods/Tanks/snippets.json]: Any other tanks configured like that?

File: data/mods/More_Survival_Tools/snippets.json (hunk 1 above)
Comment (DangerNoodle) [inline data/mods/More_Survival_Tools/snippets.json]: Not sure how to attribute it. Wiki only says it "might" be an Internet Oracle reference, and further cites how the caveman class can freely commit cannibalism.

File: data/mods/Tanks/snippets.json (hunk 2 above)
Comment (BorkBorkGoesTheCode) [inline data/mods/Tanks/snippets.json]: Not that I know of. It's done because z-level tanks aren't possible yet.

Comment (DangerNoodle) [inline data/mods/Tanks/snippets.json]: As I suspected, yes. Does seem to be the least messy layout given current restrictions.

File: data/mods/More_Survival_Tools/snippets.json (hunk 1 above)
Comment (BorkBorkGoesTheCode) [inline data/mods/More_Survival_Tools/snippets.json]: Might be good to take it out.

Comment (DangerNoodle) [inline data/mods/More_Survival_Tools/snippets.json]: Very well then. Could not think of a replacement snippet, but four more seems acceptable.

File: data/mods/FictonalWeapons/snippets.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Gun.  Sword. Gunsword.  Screw bayonets, this is way more awesome.\"" ]
```
Comment (mugling) [inline data/mods/FictonalWeapons/snippets.json]: Spacing

File: data/mods/Medieval_Stuff/snippets.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"Got every kinda sword I need, except a spatha.  And an urumi.  And maybe a szabla too.\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": [ "\"whys it called a heater shield anyway?  only heats if i light it on fire.\"" ]
```
Comment (mugling) [inline data/mods/Medieval_Stuff/snippets.json]: Capitals?

**Pull request merged**